### PR TITLE
feat(chicken-bond): Add Chicken Bond and bLUSD app token

### PR DESCRIPTION
**Pull request by wpoulin**
## Description

- Add chicken bond and bLUSD

## Checklist

- [x] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [ ] (optional) As a contributor, my Ethereum address/ENS is:
- [ ] (optional) As a contributor, my Twitter handle is:

### CODE REVIEW

File: src/apps/chicken-bond/chicken-bond.definition.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { Register } from '~app-toolkit/decorators';
+import { AppDefinition, appDefinition } from '~app/app.definition';
+import { AppAction, AppTag, GroupType } from '~app/app.interface';
+import { Network } from '~types/network.interface';
+
+export const CHICKEN_BOND_DEFINITION = appDefinition({
+  id: 'chicken-bond',
+  name: 'Chicken Bond',
+  description: `A novel bonding mechanism - first applied to Liquity LUSD.`,
+  url: 'https://botto.com/',
```
Comment (immasandwich) [inline src/apps/chicken-bond/chicken-bond.definition.ts:10]: Nope

File: src/apps/chicken-bond/chicken-bond.definition.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    discord: 'https://discord.com/invite/5HXTuqeajm',
+    twitter: 'https://twitter.com/ChickenBonds',
+  },
+
+  groups: {
+    bond: {
+      id: 'bond',
+      type: GroupType.POSITION,
+      label: 'Bond',
+    },
```
Comment (immasandwich) [inline src/apps/chicken-bond/chicken-bond.definition.ts:22]: Add the bLUSD token group here too

**Review by immasandwich — APPROVED** [review]

**Comment by bojan-liquity** [comment]
Thanks for adding Chicken Bonds but the presentation is not quite correct.
image

The red bonds have been claimed, meaning there is no LUSD value there anymore, should not be counted in the total.
The yellow bond is OK but both tokens are technically "claimable" - you can either claim back the LUSD forgo the accrued bLUSD or vice versa. So it's always either/or.

**Comment by wpoulin** [comment]
Here's the fix, https://github.com/Zapper-fi/studio/pull/1742
sorry for the inconvenience

**Pull request merged**